code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -1201,3 +1180,36 @@ func getExistingFileRef(fsh *StorageHandler, ctx context.Context, r *http.Reques
 	}
 	return nil
 }
+
+func createDir(ctx context.Context, connectionID, allocationID, clientID, dirPath string) error {
```

Comment:
hmm. it seems is not on right direction. I think we can add folders on RefTree only like '/'. and it can be saved in db.